src/styles/elements/typography.css

```diff
@@ -1 +1,2 @@
 @import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Atkinson+Hyperlegible:ital,wght@0,400;0,700;1,400;1,700&family=Newsreader:ital,opsz,wght@0,6..72,200..800;1,6..72,200..800&display=swap");
```

Comment:
More of a question than part of the review since I'm not familiar with using this. Is this a lookup to an external site? If so, how does that work if not connected to the web?